content/blog/2020-03-12-native-classroom/index.Rmarkdown

```diff
@@ -0,0 +1,93 @@
+- For the future, when students might bring in their own computers, I’m working on (and asking others to help) write bash scripts for different machines, which install R, Rstudio, and several base packages. I can distribute these bash scripts, and R scripts for the day’s exercises on thumb drives **(total Wi-Fi independence)** and have students go through dreaded installations with minimal pain. In any case, I do some intros and demos (fun) before installations begin (not fun).
+
+For **presentations**, I have a powerful mini projector with blue tooth connection (an Anker Capsule, many thanks to NIH research funding). So regardless of the awkward shape or layout of the room available to me, or however weak the projector is, I know I have my own flexible little powerhouse whose ins-and-outs I’m intimately familiar with.  
+
+There are two other workarounds (both Wi-Fi dependent) for students to watch **live coding** when visibility is poor. First is an institutional Zoom account that allows me to share my screen with several students. Another is the new `livecode` package from Colin Rundel that I’m excited to try. Check out the demo from Rstudioconf::2020 [here](https://resources.rstudio.com/rstudio-conf-2020/lightning-talk-colin-rundel). Essentially, your RStudio IDE is constantly creating a web page of your work, which your students can watch with the appropriate link.
+
+# **3. Finally, a make or break attitude for teaching beginners**
+
+We generally know, but regularly need to remember... When we first saw R or an RStudio IDE it was daunting and foreign. We may have since grown complacent and/or proud, having conquered many tasks through hours (or years) of debugging and google searches. Despite this, we can't give in to the temptation to trivialize, oversimplify, or show off skills in front of students. Our goal is to educate - not to impress and not to be lofty beacons of knowledge. In my experience, flashy shows and rapid demos often succeed at disheartening talented people. Knowledge and skills will feel unattainable to all but a very small percentage of type-A students. 
+
+So, I’ll end here. One thing I love about the RStudio community is that this reminder is rarely necessary. People I encounter whose knowledge far exceeds mine most often act with humility and respect for everyone’s stage of the journey. I’m deeply grateful to the RStudio community and education department for their support and training. From my start as a conference diversity scholar, to receiving instructor certification, to this guest blog post -- RStudio has played an enormous part of my teaching journey. Thank you, RStudio! 
```

Comment:
Remove the first sentence.

Reply:
I would replace "RStudio community" with "R community".

Reply:
I would remove "Thank you, RStudio!" from the end - might be appropriate on the author's blog, but feels self-congratulatory on ours.